src/WebJobs.Script/Workers/Rpc/MessageExtensions/RpcMessageExtensionUtilities.cs

```diff
@@ -86,9 +86,9 @@ private static SameSiteMode RpcSameSiteEnumConverter(RpcHttpCookie.Types.SameSit
                 case RpcHttpCookie.Types.SameSite.Lax:
                     return SameSiteMode.Lax;
                 case RpcHttpCookie.Types.SameSite.None:
-                    return SameSiteMode.None;
```

Comment:
If we get the protobuf message default "RpcHttpCookie.Types.SameSite.None" value, do not add a SameSite property. In V2, SameSiteMode.None (the default) becoming  would become an cookie with no samesite property set. In .NET Core 3.1, SameSiteMode.None becomes an additional property on the cookie `samesite=none`. This change preserves V2 behavior. 

The addition of `samesite=none` introduced in .NET Core 3.1 will be addressed here: https://github.com/Azure/azure-functions-nodejs-worker/pull/270

We do not have to worry about the default when cookieOptions.SameSite is not present because this is not a valid case for the cookies we build.